### PR TITLE
Webpack-todolist structure with linters and githubflow

### PR DESCRIPTION
**Pull request by kayjeee**
Dear Reviewer

Thank you for taking the time to review my pull request. I would greatly appreciate it if you could follow me,
 You can find this project here:  GitHub: [@kayjeee](https://github.com/kayjeee/webpack-todolist).
  Please leave a star it helps.  so you can receive notifications whenever I make updates to the pull request or respond to comments.

Highlights
Set up a new project with webpack that is based on the webpack exercise you have already completed.
Create an index.html file and write your HTML markup here. Created an empty To Do List placeholder . 
Created an index.js file and set an array of some simple to do tasks (array of objects). Each task object  contains three keys:
description [string].completed [bool].index: [number].
Wrote a function to iterate over the tasks array and populate an HTML list item element for each task.
On page load rendered the dynamically created list of tasks in the dedicated placeholder. The list  appears in order of the index values for each task.
Created a style.css and set rules for the To Do List. CSS must be loaded by Webpack Style/CSS Loader. the list should be a clone of the part of the minimalist project captured in the video belowhttps://www.youtube.com/watch?v=AcUd-_Yjjqg

Cheers, and Happy Reviewing!👏👏👏
Optional suggestions

 Follow me here Github:[@kayjeee ](https://github.com/kayjeee)
 Follow me here LinkedIn:[Kagiso Sebogodi](https://www.linkedin.com/in/kagiso-sebogodi/)
 Follow me here on Facebook:[kagiso sebogodi](https://www.facebook.com/kayjeee)

Thank you again for your time and consideration.

Best regards,
[@kayjeee]

### CODE REVIEW

**Comment by kayjeee** [comment]
Hi  reviewer 

With all due respect, when exactly will you give me a review?
I have made other commitments today and was hoping to get a review soon, so I can comlpete the other activities,
Getting to chat with you would be appriciated

**Review by 1GORDON — COMMENTED** [review]
##
## Hi @kayjeee ✋

Good job so far!

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

### This project will be marked as Invalid and won't affect the number of your weekly reviews.

- [x] Kindly ensure your project is using the correct [readme template](https://github.com/microverseinc/readme-template) provided in the general requirements. You may customize it accordingly and then you can resubmit your project for review.

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AlphaNtihinduka — COMMENTED** [review]
Hi @kayjeee ,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.
# Invalid Code Review Request ❌ 

You have submitted a project with a readme file that is not detailed. Kindly edit and customize your readme with the project information for it to be professional. In particular, consider adding `show support` section, add `prequisities` and finally you may consider removing the `live demo` and `loom walk through` sections since there is neither project presentation nor deployed todo list link ☺️.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready pray

Cheers and Happy coding!clapclapclap

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

**Review by IngDavidTM — APPROVED** [review]
## Hi @kayjeee  🙋‍♂️, 

Good job so far!
You have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!



## Top highlight 🎯
- The Pull Request follows the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 

## Suggested changes

- [x] While you make a great effort with your project :+1:, as a project requirement you should make sure that the list is a clone of the minimalist project captured in the [video](https://www.youtube.com/watch?v=AcUd-_Yjjqg).

Screenshot from 2023-05-10 18-17-35


- [x] _Check the comments under the review._ ⬇

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by kayjeee** [comment]
Hi @AlphaNtihinduka 

Great response and suggestions, I can't believe I forgot the readme

You also said I should pray, as highlighted in the screenshot below

screenshot alpha review

Oh boy, here is my prayer 

Nice chatting with you I am willing to follow your GitHub page and receive tips on how to be a reviewer at Microverse 
you can also add to my software journey by 
Following me here Github:[@kayjeee](https://github.com/kayjeee)
Following me here LinkedIn:[Kagiso Sebogodi](https://www.linkedin.com/in/kagiso-sebogodi/)
Following me here on Facebook:[kagiso sebogodi](https://www.facebook.com/kayjeee)

Best regards cheers @kayjeee

**Comment by kayjeee** [comment]
Hi @IngDavidTM 

Can you please be more specific 
which part exactly would you like to fix, HTML, CSS, or JS
Should I change the title on the HTML  so it says Today's todo list

**Comment by AlphaNtihinduka** [comment]
> Hi @AlphaNtihinduka 
> 
> Great response and suggestions, I can't believe I forgot the readme
> 
> You also said I should pray, as highlighted in the screenshot below
> 
> screenshot alpha review
> 
> Oh boy, here is my prayer 
> 
> Nice chatting with you I am willing to follow your GitHub page and receive tips on how to be a reviewer at Microverse 
> you can also add to my software journey by 
> Following me here Github:[@kayjeee](https://github.com/kayjeee)
> Following me here LinkedIn:[Kagiso Sebogodi](https://www.linkedin.com/in/kagiso-sebogodi/)
> Following me here on Facebook:[kagiso sebogodi](https://www.facebook.com/kayjeee)
> 
> Best regards cheers @kayjeee 
> 

Hello @kayjeee 
Thanks for reaching out to me. Sorry for the word. I was trying to use high five/pray imoji😁🙈. It will be wonderful connecting with you.
Happy coding.

File: src/style.css [hunk 1]
```diff
@@ -1,3 +1,146 @@
...
+  justify-content: flex-start;
+  gap: 20px;
+  position: relative;
+  width: 100%;
+  margin-block: 10px;
+  border-bottom: 1px solid gray;
+  border-top: 1px solid gray;
+  padding-block: 5px;
+  animation: fade-in 0.5s ease-in;
+}
```
Comment (Shedrack-Sunday) [inline src/style.css]: - [ ] You did well 👏 . But as stated by the previous reviewer, follow the project requirement and make sure that the list is a clone of the minimalist project captured in the [video](https://www.youtube.com/watch?v=AcUd-_Yjjqg).
- I can see added lines in your UI compared to the simple yet well to do design given

## Your work
image
 
 ## Design given:
image

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hello @kayjeee        👋 

# CHANGES REQUIRED ♻️

Great work on the project 👏
You've done well so far in the project,  However, there are a few you need to work on, to move to the next project. 💯

 


## Highlights      
- Moderate UI so far 👍  

  
# Required Changes  🌵 

- [x] You made some impressive implementations based on the project requirements as asked by the last reviwer, but there's just one more you need to adjust

_Check the comments for a detailed review 

# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/style.css (hunk 1 above)
Comment (kayjeee) [reply src/style.css]: Hi  @Shedrack-Sunday 

Thank you for the good feedback
I removed the lines as requested

**Review by divinecharlotte — APPROVED** [review]
# Hello @kayjeee  😃 🥳 🎉 🎈 

Great Job 😄 👏 You've done it. Your project is complete!  There is nothing else to say  👍🏻 ✅ 
### To Highlight 💯

- Descriptive pr👍 
- linters are passing 👍 
 




## Congratulations! 🎉 



Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by kayjeee** [comment]
Dear @divinecharlotte ,

I just wanted to take a moment to express my gratitude for your recent review feedback on my work. Your valuable comments and suggestions have been extremely helpful in improving my work and making it better.

I appreciate the time and effort you took to carefully review my work and provide such insightful feedback. Your approval of my work means a lot to me and gives me the confidence to continue to strive for excellence.

Thank you once again for your support and encouragement. Your contributions have made a significant impact on my work, and I am truly grateful.

Nice chatting with you I am willing to follow your GitHub page and receive tips on how to be a reviewer at Microverse
you can also add to my software journey by
Following me here Github:[@kayjeee](https://github.com/kayjeee)
Following me here LinkedIn:[Kagiso Sebogodi](https://www.linkedin.com/in/kagiso-sebogodi/)
Following me here on Facebook:[kagiso sebogodi](https://www.facebook.com/kayjeee)

Best regards,
[@kayjeee ]

**Pull request merged**